project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
> why we shouldn't use it

Only my opinion:
 - increase your compile time
 - add [accidental complexity](https://en.wikipedia.org/wiki/No_Silver_Bullet) (new dsl, macro, new library) that does not solve any problem (you **should** see any compilation error without it)